improver_tests/acceptance/SHA256SUMS

```diff
@@ -584,37 +584,37 @@ e653ae271dd789dde8b03e6237cc7b48552dd9d5206d3430ee68d283683247a0  ./wind_downsca
-431a9f3cedf30b2f71ed23fc25b53c92d4261dbf78e3703eeb919cf426af238c  ./wxcode/global/probability_of_low_and_medium_type_cloud_area_fraction_above_threshold.nc
-ef726aa712573f5c67b91fa347759d1d27f931577d1dadb7d524445dcb140d15  ./wxcode/global/probability_of_low_type_cloud_area_fraction_above_threshold.nc
-bfb5ca02da527484121636a7189a42103793a6848003b1bb0f78ff70955b50d3  ./wxcode/global/probability_of_lwe_precipitation_rate_above_threshold.nc
-cb33e5a50759547a35bb5688b7bef7942cd9693f967cd2fc88a97357735fe759  ./wxcode/global/probability_of_lwe_precipitation_rate_in_vicinity_above_threshold.nc
-4e2572dbae885b8c041944f430dc6db2d58c281fdd2f53d7910e53e22af7783a  ./wxcode/global/probability_of_lwe_sleetfall_rate_above_threshold.nc
-142e5163a2cf3c56976786b9caa788ca0e15dce3e2c64f2013d72c96e171a3e4  ./wxcode/global/probability_of_lwe_snowfall_rate_above_threshold.nc
-1d9ed8128d5870a23115e8407bebac9444646a7d2212f8ff8f572d7da30c97ff  ./wxcode/global/probability_of_rainfall_rate_above_threshold.nc
-2fc6a941573c143f523707439e81a8500c24acea48bdd327bd00244d26f6c4e8  ./wxcode/global/probability_of_shower_condition_above_threshold.nc
-1ed1a279a522a78e1a38f715e6a2a72de795389ee35a15c1db9bbf23bcd23872  ./wxcode/global/probability_of_visibility_in_air_below_threshold.nc
-c6cd3558047fb7e32f0b153615a479e7913f071b9050002f8bcb44562e8b200f  ./wxcode/native_units/probability_of_lightning_flashes_per_unit_area_in_vicinity_above_threshold.nc
-9abb0bd5079f5e8b127fca98605791cc3d8655be24316a91e419f6782b38d5a0  ./wxcode/native_units/probability_of_low_and_medium_type_cloud_area_fraction_above_threshold.nc
-642e08eb34a3f37e2219520d71eafaeaec27691bdb5c5a91d9aa1fa2cc5fd1ab  ./wxcode/native_units/probability_of_low_type_cloud_area_fraction_above_threshold.nc
-5927c86c42d3f5d160de851d822cbe29c59d1c2458f531e3dab03a133221f286  ./wxcode/native_units/probability_of_lwe_precipitation_rate_above_threshold.nc
-e47fdbc521c8dc8501d72cfa46981e3072b8aa7b0d27eed97b73a14b3a47e803  ./wxcode/native_units/probability_of_lwe_precipitation_rate_in_vicinity_above_threshold.nc
-ee6f2f7b04eb8f77274ea5aa10ad0b58f0cb3cd323d4582accb1e34e964fb67c  ./wxcode/native_units/probability_of_lwe_sleetfall_rate_above_threshold.nc
-d134182bb9c7b9057bd2fb53fd7c3e4bc6acbbfbf4e66b19d538d6ddd0612fe5  ./wxcode/native_units/probability_of_lwe_snowfall_rate_above_threshold.nc
-05f7b6d9d5d70ebb8474701ee2b0988371cfdd64626f66e1c74375d52a4e77de  ./wxcode/native_units/probability_of_rainfall_rate_above_threshold.nc
-4a9e76b9cc2be07c7866da978918e57814e4469f249492e60113000cf3d60c15  ./wxcode/native_units/probability_of_shower_condition_above_threshold.nc
-66b7baead79b9bf85184af3276b1cc0f3c76b9e94d88534d440e14268b37565b  ./wxcode/native_units/probability_of_visibility_in_air_below_threshold.nc
-94c3674e6d690a5fe1379924f30e0dbaf35a1aeabcf24fc251c3b7d1989fe507  ./wxcode/wx_decision_tree.json
+a10668de8e2e0f506f4001eff6290adb3bc98eef49fd4a3eb304b166a60fe0ce  ./wxcode/basic/kgo.nc
```

Comment:
Note that I'm getting acceptance test failures due to checksum failures. The tests pass otherwise.

Reply:
Fixed.